### PR TITLE
[Doc](map-type) add map sql doc 

### PR DESCRIPTION
**Pull request by amorynan**
# Proposed changes
add map manual doc 

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Comment by dataroaring** [comment]
english version is also expected.

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## MAP
+
+### name
+
+<version since="1.2.0">
```
Comment (xiaokang) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:31]: version 2.0.0

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+MAP
+
+</version>
+
+### description
+
+`MAP<T1, T2>`
```
Comment (xiaokang) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:39]: MAP<K, V>

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            ) ENGINE=OLAP
+            DUPLICATE KEY(`k1`)
+            DISTRIBUTED BY HASH(`k1`) BUCKETS 1
+            PROPERTIES (
+            "replication_allocation" = "tag.location.default: 1",
+            "storage_format" = "V2"
+            )
+```
+
+插入数据示例：
```
Comment (xiaokang) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:67]: add example for stream load

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+mysql> SELECT m['a'] FROM simple_map;
++-----------------------------+
+| %element_extract%(`m`, 'a') |
++-----------------------------+
+|                         100 |
+|                        NULL |
+|                          10 |
++-----------------------------+
+```
+
```
Comment (xiaokang) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:111]: add examples for map functions

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+---
+{
+"title": "MAP",
+"language": "zh-CN"
```
Comment (xiaokang) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:4]: add english doc and side bar

Comment (xiaokang) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Types/MAP.md:4]: in docs/sidebars.json

**Comment by xy720** [comment]
can you please add english docs?

**Comment by amorynan** [comment]
> 

Done ! please review it !

**Comment by amorynan** [comment]
run buildall

**Review by xy720 — DISMISSED** [review]
LGTM

**Comment by amorynan** [comment]
@xy720  please approve it

**Review by xy720 — APPROVED** [review]

**Comment by amorynan** [comment]
@xy720  done!

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**